mass_mailing_switzerland/models/res_partner.py

```diff
@@ -12,14 +12,20 @@
 from odoo import models, api, fields
 from odoo.http import request
 
-
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
+    def _default_odoo_list_ids(self):
+        default_val = self.env['mailchimp.lists'].search([])
+        return default_val
+
+    odoo_list_ids = fields.Many2many('mailchimp.lists', string='MailChimp Lists', default=_default_odoo_list_ids)
```

Comment:
Isn't there an other way to retrieve a partner link to a mailchimp list ?
This could be a potential duplicate